packages/components/organisms/f-content-cards/src/components/cardTemplates/StampCard1.vue

```diff
@@ -67,11 +77,35 @@
 
 <script>
 import parseISO from 'date-fns/parseISO';
-import lightFormat from 'date-fns/format';
+import format from 'date-fns/format';
+import lightFormat from 'date-fns/lightFormat';
 
 import EmptyStamp from './images/stamp-empty-15.svg';
 import FullStamp from './images/stamp-full-15.svg';
 
+import '../MakeTextAccessible';
+
+const getDateFnsLocale = locale => {
```

Comment:
Would be good to add a JSDoc for this function

Reply:
Also – any reason this isn't just included as part of `methods` rather than as a standalone function?

Reply:
no current reason so will move and add jsdoc